src/components/reapit-logo/reapit-logo.tsx

```diff
@@ -0,0 +1,8 @@
+import reapitLogoUrl from './icons/brand-reapit.svg'
+import type { ImgHTMLAttributes } from 'react'
+
+interface ReapitLogoProps extends Omit<ImgHTMLAttributes<HTMLImageElement>, 'src' | 'srcSet'> {}
+
+export const ReapitLogo = (props: ReapitLogoProps) => {
+  return <img {...props} src={reapitLogoUrl} />
```

Comment:
seemingly looks ok without additional aspect ratio adjustment, so I think I don't need to make it `styled` for now 🤔